trymito.io/components/CTAButtons/CTAButtons.tsx

```diff
@@ -10,7 +10,8 @@ const CTAButtons = (props: {
     variant: 'download' | 'contact' | 'try jupyterlite',
     align: 'left' | 'center',
     displaySecondaryCTA?: boolean
-    secondaryCTA?: 'pro' | 'learn more'
+    secondaryCTA?: 'pro' | 'learn more',
+    ctaText?: string,
```

Comment:
`ctaText` and `variant` are conflicting types -- which results in you needing to support `props.ctaText || ` in multiple places. 

Let's just change the type of `variant` to be `download | contact | try | string` -- and then pretty much add an `else` case where if it's none of the three, we display the cta text.

Reply:
They aren't the same thing because the variant specifies where the CTA actually goes, ie: go to the download page. I'm going to leave this for now.